src/agent/unified-agent.ts

```
@@ -89,13 +96,13 @@ export class UnifiedAgent implements Agent {
         case 'simple':
           result = await this.executeSimple(input, options);
           break;
-        
+
         case 'react':
           const reactResult = await this.executeReAct(input, options);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)